pkg/observ/observ.go

```diff
@@ -95,6 +98,20 @@ func registerStackdriverExporter(projectID, ENV string) (func(), error) {
 	return ex.Flush, nil
 }
 
+func registerAppInsightsExporter(endpoint, svcName, ENV string) (func(), error) {
+	const op errors.Op = "registerAppInsightsExporter"
+	ex, err := ocagent.NewExporter(
+		ocagent.WithInsecure(),
```

Comment:
Can we add a todo to remove WithInsecure() in the future

Reply:
if you'll add a TODO please create a backing task for doing this using the same pattern we have in mongo. except this would be grpc not http call